astropy/coordinates/solar_system.py

```diff
@@ -43,28 +43,42 @@
 SOLAR_SYSTEM_BODIES = tuple(BODY_NAME_TO_KERNEL_SPEC.keys())
 
 
-def _download_spk_file(url=('http://naif.jpl.nasa.gov/pub/naif/'
-                            'generic_kernels/spk/planets/de430.bsp'),
-                       show_progress=True):
+def set_kernel_url(url='http://naif.jpl.nasa.gov/pub/naif/generic_kernels'
```

Comment:
In that case, shouldn't it use the ScienceState class? (http://docs.astropy.org/en/stable/api/astropy.utils.state.ScienceState.html#astropy.utils.state.ScienceState)